### PR TITLE
Pull to refresh component

### PR DESCRIPTION
**Pull request by Globalika**
added refresh controller

### CODE REVIEW

**Review by khrystyna-sb — APPROVED** [review]

File: Countries/FirstScreen/ListViewController.swift [hunk 1]
```diff
@@ -15,14 +15,30 @@ class ListViewController: UITableViewController {
...
     var countries: [CountriesApiQuery.Data.Country] = []
     
+    let refrechControll: UIRefreshControl = {
+        let refrechControll = UIRefreshControl()
+        return refrechControll
+    }()
+
+    @objc private func refresh(sender: UIRefreshControl) {
+        loadData()
+        sender.endRefreshing()
```
Comment (apolwork) [inline Countries/FirstScreen/ListViewController.swift]: I would suggest to move end refreshing to place where you updates your data

File: Countries/FirstScreen/ListViewController.swift [hunk 2]
```diff
@@ -15,14 +15,30 @@ class ListViewController: UITableViewController {
...
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        setupRefreshController()
```
Comment (apolwork) [inline Countries/FirstScreen/ListViewController.swift:27]: It is not an issue, but I would prefer to set refresh control when you already setup your table view

**Review by apolwork — COMMENTED** [review]
Minor comments, just review them

File: Countries/FirstScreen/ListViewController.swift [hunk 3]
```diff
@@ -15,14 +15,29 @@ class ListViewController: UITableViewController {
...
-        setupNavigationItem()
+        
         registerTableView()
+        setupRefreshController()
+        setupNavigationItem()
         loadData()
     }
     
+    func setupRefreshController() {
+        tableView.refreshControl = refrechControll
```
Comment (apolwork) [inline Countries/FirstScreen/ListViewController.swift]: minor: Double L in refresh control property name

File: Countries/FirstScreen/ListViewController.swift [hunk 4]
```diff
@@ -15,14 +15,29 @@ class ListViewController: UITableViewController {
...
     var countries: [CountriesApiQuery.Data.Country] = []
     
+    let refrechControll: UIRefreshControl = {
+        let refrechControll = UIRefreshControl()
+        return refrechControll
+    }()
+
+    @objc private func refresh(sender: UIRefreshControl) {
+        loadData(sender: sender)
+    }
```
Comment (apolwork) [inline Countries/FirstScreen/ListViewController.swift]: Do you need this function, as it only pass argument to the next one?

Comment (Globalika) [reply Countries/FirstScreen/ListViewController.swift]: yes.. i mean i dont need it at all

**Review by apolwork — APPROVED** [review]

**Pull request merged**